testrunner/tests/media.spec.js

```diff
@@ -302,56 +302,56 @@ QUnit.test('Get number of available bytes from storage device', function(assert)
       throw new Error('Failed to get initial free bytes from ' + storage.storageName);
     }
 
-   //  var mutableSize = 4097; 
-   //  var content = ''; 
-   //  for(var i = 0 ; i < mutableSize; i++ ) {
-   //    content += '1';  
-   //  }
-    
+    //  var mutableSize = 4097; 
+    //  var content = ''; 
+    //  for(var i = 0 ; i < mutableSize; i++ ) {
+    //    content += '1';  
+    //  }
+
     var content = '1234567890';
-    
-    var file = new File([content], 'Size'+content.length+'.txt', {
-        type: 'text/plain'
+
+    var file = new File([content], 'Size' + content.length + '.txt', {
+      type: 'text/plain'
     });
-    
+
     // free bytes before writing file
     startFreeBytes = bytesBefore;
-    
+
     //delete last generated test*size.txt file before testing.
-    
+
     var resource = navigator.getDeviceStorages('sdcard')[1];
     var request = resource.delete('backup/test' + file.name);
-    
+
     request.onerror = function() {
-      alert(this.error.name + 'Failed to remove last generated test file' );
-    }
-    
+      alert(this.error.name + 'Failed to remove last generated test file');
```

Comment:
I generally prefer not to have alerts in specs so that a million modals don't pop up